### PR TITLE
PostView feature

### PR DESCRIPTION
**Pull request by spunzh**
Changed display logic of cells
Created 4 different post cells

### CODE REVIEW

**Comment by spunzh** [comment]
https://user-images.githubusercontent.com/42088475/132305400-27691cb7-66fb-4154-97b1-8d0ce4ffaaf4.mp4

**Comment by spunzh** [comment]
https://user-images.githubusercontent.com/42088475/132313584-6ecae045-c5d9-41ae-a605-0e3f98ec8c21.mp4

File: VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostHeaderTableViewСell.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
+// PostHeaderTableViewСell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+final class PostHeaderTableViewСell: UITableViewCell {
+    // MARK: - IBOutlets
+
+    @IBOutlet var avatarImageView: AvatarImageCustomView!
```
Comment (iMrOwl) [inline VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostHeaderTableViewСell.swift:9]: приват

File: VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostHeaderTableViewСell.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    @IBOutlet var userNameLabel: UILabel!
+    @IBOutlet var postDateLabel: UILabel!
+
+    // MARK: - AwakeFromNib
+
+    override func awakeFromNib() {
+        userAvatarImageSetup()
```
Comment (iMrOwl) [inline VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostHeaderTableViewСell.swift:16]: супер?

File: VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostHeaderTableViewСell.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+
+    // MARK: - Public Methods
+
+    public func fill(with info: Post) {
+        userNameLabel.text = info.userName
+        postDateLabel.text = info.date
+        guard let avatar = UIImage(named: info.userAvatar)
```
Comment (iMrOwl) [inline VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostHeaderTableViewСell.swift:24]: в одну

File: VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostPhotoTableViewCell.swift [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+final class PostPhotoTableViewCell: UITableViewCell {
+    // MARK: - IBOutlets
+
+    @IBOutlet private var userPostImage: UIImageView!
+
+    // MARK: - Public Methods
+
+    public func fill(with post: Post) {
+        guard let postImage = UIImage(named: post.photo)
```
Comment (iMrOwl) [inline VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostPhotoTableViewCell.swift:14]: в одну

File: VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostTextTableViewCell.swift [hunk 5]
```diff
@@ -0,0 +1,16 @@
+// PostTextTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+final class PostTextTableViewCell: UITableViewCell {
+    // MARK: - IBOutlets
+
+    @IBOutlet var userPostLabel: UILabel!
```
Comment (iMrOwl) [inline VKClient/VKClient/ViewControllers/NewsViewController/Cell/PostTextTableViewCell.swift:9]: приват

**Review by iMrOwl — APPROVED** [review]

**Pull request merged**